docs/azure/configure-vs-code.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Configure Visual Studio Code for Azure development with .NET
 description: This article helps you configure Visual Studio Code for Azure development including getting the right plugins installed and configured in VS Code
-ms.topic: conceptual
+ms.topic: concept-article
```

Comment:
I would be cautious about changing things to **concept-article**. That has a schema associated with it, which for example, disallows numbered lists. This article contains a numbered list and thus would be in violation of the article-type schema.